### PR TITLE
Remove some stuff from util

### PR DESCRIPTION
**Pull request by nox**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/12224)

### CODE REVIEW

**Comment by frewsxcv** [comment]
@bors-servo r-

```
./components/style/lib.rs:45: extern crate declaration is not in alphabetical order
	expected: euclid
	found: deque
```

**Comment by frewsxcv** [comment]
Don't blame me @bors-servo, I told you to stop before you even started. Don't even go there.

File: components/style/workqueue.rs [hunk 1]
```diff
@@ -7,6 +7,8 @@
 //! Data associated with queues is simply a pair of unsigned integers. It is expected that a
 //! higher-level API on top of this could allow safe fork-join parallelism.
 
+#![allow(unsafe_code)]
+
 #[cfg(windows)]
 extern crate kernel32;
```
Comment (jdm) [inline components/style/workqueue.rs]: Cargo.toml needs to be adjusted.

**Pull request merged**